docs/content/preview/drivers-orms/nodejs/sequelize-yugabytedb.md

```diff
@@ -180,6 +180,31 @@ Employees Details:
 ]
 ```
 
+## Using Cloud Cluster to connect with Seqeulize:
```

Comment:
Can this be something like "Specifying SSL configuration"?
And then we can mention that this is required if one is connecting to a YB Managed cluster or even a local YB cluster with SSL enabled.